### ISSUE TITLE
Priority 1 accessibility issue: PDF documents did not have a title.

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6538]
**What**
The PDF document is not titled. This means screen reader users are not presented with the document purpose and must rely on the filename, which is ambiguous.

**WCAG Reference:**
2.4.2 Page Titled (Level A)
Understanding Page Titled | How to Meet Page Titled

**Low vision comments:**
“When I wanted to read through the PDF document, I noticed that the title of the PDF is not visible, this means that users like myself won’t be able to clearly identify what page we’re on nor will we be able to identify what the title of the PDF document is which may stop viewers to continue reading.”
 
Solution:
Ensure that all documents have a descriptive title that describes context of the document. For more information, please visit Techniques for WCAG 2.1 PDF18: Specifying the document title using the Title entry in the document information dictionary of a PDF document.

**Where**

<img width="996" alt="Screenshot 2021-05-27 at 10 43 28" src="https://user-images.githubusercontent.com/7315126/119804618-6b756080-bed8-11eb-8dfc-df91e8c1d602.png">

**Comment by dannybirchall** [comment]
I suspect that this is not the only case, and that all our PDFs could be audited. 

... and also that we might check for other aspects of accessibility

Prismic's image library won't allow us to filter/identify PDFs

Is there any quick way of identifying a list of PDFs on the site and/or checking which elements are missing?

**Comment by gestchild** [comment]
@dannybirchall I don't think there is an easy way to list all the PDFs in the Prismic media library, so I loaded all the docs in the browser then scraped the html (this was done @ 14:40ish 01/07/2021). There were 62 pdfs listed in total:

Wellcome Collection Venue Hire Terms and Conditions.pdf
Wellcome Collection Access Procedures Aug 2020.pdf
access-to-personal-data.pdf
libraryapplication.pdf
Perspectives_on_play_learning_pack.pdf
access-to-personal-data.pdf
Ts and Cs - September 2020.pdf
RawMinds changing our planet.pdf
WellcomeCollection_RawMinds_BeingUs.pdf
Wellcome Collection_Access Policy_Aug 2020.pdf
Facilitator brief – Ideas Hub at Wellcome Collection.pdf
Publishing Development Initiative.pdf
Publishing Writers Development Brief.pdf
Terms and Conditions 2020.pdf
WC Digitisation Strategy 2020-2025.pdf
Wellcome Collection Menu 2020.pdf
Wellcome_K_Children_Menu_A5_Oct_19.pdf
Wellcome_K_Menu_folded_296x250mm_Oct_19_v2.pdf
2019-10-18-glam-cam-digipres-slides.pdf
Forward Plan 19.pdf
Digital Preservation Policy.pdf
WC Venue Hire Autumn Winter 2019 Menu.pdf
List of human remains in Sir Henry Wellcomes Museum Collection.pdf
Wellcome_Menu_148x250mm_4pp_folded_April_19_web_2.pdf
Event Spaces Menu Spring 2019.pdf
Event Spaces Menu Spring 2019.pdf
Wellcome Collection Research Development Strategy.pdf
Wellcome_Afternoon_Tea_Oct2018_A5_4pp.pdf
Wellcome Kitchen Christmas menu 2018.pdf
Wellcome_Food_&amp;_drinks_Oct2018_4pp_148x250mm (3).pdf
Wellcome_Teeth_Food_&amp;_drinks_May2018_A5.pdf
Wellcome Collection Event Spaces Food and Drink Menu.pdf
Med Now Res_FULL.pdf
Med Man Res_FULL.pdf
Med Now Res_POST16 final.pdf
Med Now Res_KS4 final.pdf
Med Now Res_TRAIL.pdf
Med Now Res_INTRO.pdf
Med Now Res_LINKS.pdf
Med Man Res_TRAIL2.pdf
Med Man Res_TRAIL1.pdf
Med Man Res_TRAIL3.pdf
Med Man Res_INTRO.pdf
Med Man Res_LINKS.pdf
Wellcome_Drinks_March2018_A5_2pp.pdf
Wellcome_Afternoon_Tea_Mar2018_A5web.pdf
2018-08-15 Wellcome Collection Access Diversity Inclusion Strategy .pdf
Wellcome Collection Event Spaces_online brochure.pdf
Wellcome_Teeth_Food_&amp;_drinks_May2018_A5_4pp.pdf
Wellcome Collection Who we are and what we do_2.pdf
Wellcome Collection_Collections Development Policy_March 2018.pdf
Wellcome Collection_Collections Information Policy_March 2018.pdf
Wellcome Collection_Conservation and Collections Care Policy_March 2018.pdf
Wellcome Collection_Forward Plan_2018-23.pdf
School events brochure Jan-July 2018 (967KB).pdf
Wellcome Collection Venue Hire Terms &amp; Conditions (114 KB).pdf
Event Spaces Menu April 2018.pdf
MP-6761.23_A-Man_Large_Print_Guide_Exhibit_Texts_reproduction_web.pdf
MP-6761.23_A-Man_Large_Print_Guide_Exhibit_Texts_AW.pdf
MP-6761.23_A-Man_Large_Print_Guide_Correspondence_AW.pdf
CGDSYL-LargePrintGuide.pdf
bedlam_large_text_guide.pdf

**Comment by dannybirchall** [comment]
Thanks @gestchild !

**Comment by pollecuttn** [comment]
https://wellcome.slack.com/archives/C3N7J05TK/p1625656260067200

"Danny has clarified (with Youth Programmes I think) what is required to make an accessible PDF and they are now going to be supplied properly formatted so they are accessible. I think this was linked with work going on more broadly in the Trust / Collection... maybe this: https://trustnet.wellcome.org/news/brief-guide-publishing-accessible-documents"

"I think the Y[outh] P[rogrammes] PDFs are adding new content"

To be followed up by @dannybirchall 's team

**Comment by jennpb** [comment]
@dannybirchall @pollecuttn Can I check the progress on this, and any decisions made over the past year?
